recipes/approvaltests.cpp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,9 @@
+#include <cstdlib>
+#include <iostream>
+#include "ApprovalTests.hpp"
+
+
+int main(void) {
+    std::cout << ApprovalTests::StringMaker::toString(42) << " is the answer."<< std::endl;
```

Comment:
```suggestion
    std::cout << ApprovalTests::StringMaker::toString(42) << ApprovalTests::Grid::print(1, 1, " is the answer.")<< std::endl;
```
ApprovalTests::Grid::print is not inline, so we are sure that the library is not discarded